kurl_util/cmd/toml/main_test.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/pelletier/go-toml"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListLeaves(t *testing.T) {
+	var treeString = `
+[plugins]
+  [plugins."io.containerd.grpc.v1.cri"]
+    systemd_cgroup = false
+    [plugins."io.containerd.grpc.v1.cri".containerd]
+      [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
+        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+          runtime_type = "io.containerd.runc.v1"`
+	tree, err := toml.Load(treeString)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	expect := [][]string{
+		{"plugins", "io.containerd.grpc.v1.cri", "containerd", "runtimes", "runc", "runtime_type"},
+		{"plugins", "io.containerd.grpc.v1.cri", "systemd_cgroup"},
+	}
+	actual := listLeaves(tree)
+
+	for _, leaf := range expect {
+		assert.Contains(t, actual, leaf)
+	}
+}
+
+func TestMerge(t *testing.T) {
+	base := `
+version = 2
+
+[debug]
```

Comment:
What happens if `debug` doesn't exist? Can we test that (patching in a new value instead of changing a value) too?